src/PKSim.Core/Snapshots/Mappers/ClassificationSnapshotTask.cs

```diff
@@ -7,22 +7,24 @@
 using ModelClassification = OSPSuite.Core.Domain.Classification;
 using SnapshotClassification = PKSim.Core.Snapshots.Classification;
 
-
 namespace PKSim.Core.Snapshots.Mappers
 {
    public interface IClassificationSnapshotTask
    {
       /// <summary>
-      ///   Maps the <paramref name="snapshots"/> into classifications of type <paramref name="classificationType"/> and adds them to the <paramref name="project"/>.
-      ///   Also added to the project will be classifiables contained in the classifications and those will have the subjects
-      ///   configured correctly from the list of <paramref name="subjects"/>.
+      ///    Maps the <paramref name="snapshots" /> into classifications  and adds them to the <paramref name="project" />.
+      ///    Also added to the project will be classifiables contained in the classifications and those will have the subjects
+      ///    configured correctly from the list of <paramref name="subjects" />.
       /// </summary>
-      /// <typeparam name="TClassifiable">This is the type of classifiable that is created for each classifiable in each newly created classification /></typeparam>
+      /// <typeparam name="TClassifiable">
+      ///    This is the type of classifiable that is created for each classifiable in each newly
+      ///    created classification />
+      /// </typeparam>
       /// <typeparam name="TSubject">This is the type of the subject for the classifiable</typeparam>
-      Task UpdateProjectClassifications<TClassifiable, TSubject>(SnapshotClassification[] snapshots, PKSimProject project, IReadOnlyCollection<TSubject> subjects, ClassificationType classificationType) 
+      Task UpdateProjectClassifications<TClassifiable, TSubject>(SnapshotClassification[] snapshots, PKSimProject project, IReadOnlyCollection<TSubject> subjects)
          where TClassifiable : Classifiable<TSubject>, new() where TSubject : IObjectBase;
 
-      Task<SnapshotClassification[]> MapClassificationsToSnapshots(IReadOnlyList<ModelClassification> classifications, IReadOnlyList<IClassifiableWrapper> classifiables);
+      Task<SnapshotClassification[]> MapClassificationsToSnapshots<TClassifiable>(PKSimProject project) where TClassifiable : class, IClassifiableWrapper, new();
```

Comment:
That function needed to be a one liner since we will have to duplicate the code for all classification.